docs/guides/admin/docs/workflowoperationhandlers/changetype-woh.md

```diff
@@ -0,0 +1,45 @@
+Clone Workflow Operation
+========================
+
+ID: `changetype`
+
+Description
+-----------
+
+The change type workflow operation can be used to change the type of media package elements.
+
+
+Parameter Table
+---------------
+
+|Configuration Key         |Description                                       |Example           |
+|--------------------------|--------------------------------------------------|------------------|
+|source-flavors            |The source flavor(s) to clone                     |presenter/source  |
+|source-tags               |Comma-separated list of source-tags               |archive           |
+|target-flavor\*           |The target flavor                                 |presenter/target  |
+|target-type\*             |The target type                                   |track             |
+
+\* mandatory configuration key
+
+Notes:
+
+- *source-flavor* and *source-tags* may be used both together to select media package elements based on both flavors and
+  tags If *source-flavor* is not specified, all media package elements matching *source-tags* will be selected
+- In case that neither *source-flavor* nor *source-tags* are specified, the operation will be skipped
+- In case no media package elements match *source-flavor* and *source-tags*, the operation will be skipped
+
```

Comment:
Needs to show the valid track types. What types you can use?

Reply:
Here should be described that the types are `catalog` , `attachment` and `track`?